hl7/4.2/modules/ROOT/pages/index.adoc

```diff
@@ -109,553 +60,36 @@ sample documents. It is also possible to use an overlay schema for
-			<ee:message >
-				<ee:set-payload ><![CDATA[%dw 2.0
-output application/java
----
-{
-	Delimiters: payload.Delimiters,
-	Id: payload.Id
-}]]></ee:set-payload>
-			</ee:message>
-		</ee:transform>
-		<hl7:write doc:name="Write" config-ref="HL7_Extension_Config"/>
-	</flow>
-</mule>
-----
-
-=== Set Your HL7 Identification in XML
-
-You can configure the Message Header (MSH) application and facility identification for you and your trading partner on the HL7 EDI connector configuration.
-
-The values you set are used when writing HL7 messages to supply the namespace ID, universal ID, and universal ID type, and are verified in receive messages. If you don't want to restrict incoming messages you can leave these blank, and set the values for outgoing messages on the write operation or the actual outgoing message. Values set on the write operation override the connector configuration, and values set directly on the message override both the connector configuration and any values set on the write operation.
-
-* Self identification parameters identify your side of the trading partner relationship.
-+
-Self identification parameters:
-+
-[source,text,linenums]
-----
-appNamespaceIdSelf="<value>"
-appUniversalIdSelf="<value>"
-appUniversalIdTypeSelf="<value>"
-----
-+
-* Partner identification parameters identify your trading partner.
-+
-Partner identification parameters:
-+
-[source,text,linenums]
-----
-appNamespaceIdPartner="<value>"
-appUniversalIdPartner="<value>"
-appUniversalIdTypePartner="<value>"
-----
-
-=== Set Parser Options
-
-You can set the following options if needed:
-
-[%header%autowidth.spread]
-|===
-|XML Value |Visual Studio Option
-|eventMessageMap="/hl7/v2_5_1/event-message.yaml" |Event and message to message structure map path (required if using multiple message structures, unless the MSH-09 and MSG-03 message structure values are always present in the received messages).
-|genericExtensionPattern="Z.." |Java regular pattern for generic extension segment names (to allow handling of extension segments as maps of field values).
-|invalidCharacterInValueFail="true" |Fail when an invalid character is in a value.
-|missingRequiredValueFail="true" |Fail when a required value is missing.
-|processingId="PRODUCTION" |Required processing ID (to specify a particular processing ID required on receive messages, such as `P` for Production).
-|segmentOutOfOrderFail="true" |Fail when a segment is out of order in a message.
-|unknownSegmentFail="true" |Fail when an unknown segment is in a message.
-|unusedSegmentPresentFail="true" |Fail when an unused segment is included in a message.
-|validateHL7Version="true" |Validate HL7 Message Version.
-|valueLengthErrorFail="true" |Fail when a value length is outside an allowed range.
-|wrongSegmentsRepeatsFail="true" |Fail when there are too many repeats of a segment.
-|wrongValuesRepeatsFail="true" |Fail when there are too many repeats of a value.
-|===
-
 === Set Your Schema Locations
```

Comment:
This probably should be somewhere else. The index shouldn't have procedures. You can assess when you come back to this in a later cycle.

Reply:
moved to add config topic